src/audio/google_rtc_audio_processing.c

```diff
@@ -98,6 +98,19 @@ static int google_rtc_audio_processing_reconfigure(struct comp_dev *dev)
 
 	comp_dbg(dev, "google_rtc_audio_processing_reconfigure()");
 
+	if (!comp_is_current_data_blob_valid(cd->tuning_handler) &&
+	    !comp_is_new_data_blob_available(cd->tuning_handler)) {
+		/*
+		 * The data blob hasn't been available once so far.
+		 *
+		 * This looks redundant since the same check will be done in
+		 * comp_get_data_blob() below. But without this early return,
+		 * hundreds of warn message lines are produced per second by
+		 * comp_get_data_blob() calls until the data blob is arrived.
+		 */
+		return 0;
```

Comment:
are we sure we wouldn't be hiding a problem here? Maybe it would be better to do something like "warn once" or rate-limited warning?